examples/decoding/plot_decoding_csp_timefreq.py

```diff
@@ -0,0 +1,111 @@
+"""
+============================================================================
+Decoding in time-frequency space data using the Common Spatial Pattern (CSP)
+============================================================================
+
+The time-frequency decomposition is estimated by iterating over raw data that
+has been band-passed at different frequencies. This is used to compute a
+covariance matrix over a rolling time-window and extract the CSP filtered
+signals. A linear discriminant classifier is then applied to these signals.
+"""
+# Authors: Laura Gwilliams <laura.gwilliams@nyu.edu>
+#          Jean-Remi King <jeanremi.king@gmail.com>
+#          Alex Barachant <alexandre.barachant@gmail.com>
+#          Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from mne import Epochs, find_events, create_info
+from mne.io import concatenate_raws, read_raw_edf
+from mne.datasets import eegbci
+from mne.decoding import CSP
+from mne.time_frequency import AverageTFR
+
+from sklearn.discriminant_analysis import LinearDiscriminantAnalysis
+from sklearn.model_selection import TimeSeriesSplit, cross_val_score
+from sklearn.pipeline import make_pipeline
+
+# #############################################################################
+
```

Comment:
It doesn't quite render the sections correctly, I think because these breaks are not formatted properly

https://4168-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_examples/decoding/plot_decoding_csp_timefreq.html

I think there should be no empty newline here, and that the `# #####` above should be `######` (no space), but check existing examples to make sure